File: evals/behavioral/tasks/rust/fixtures/safe-index.good.rs
// Total: out-of-bounds is widened into Option, so the caller must handle the
// missing element. `get(&v, 99) + 1` does not type-check (Option has no Add).
pub fn get(items: &[i32], index: usize) -> Option<i32> {
    items.get(index).copied()
}

File: evals/behavioral/tasks/python/fixtures/safe-index.good.py
# Total: out-of-bounds is widened into `int | None`, so the caller must handle
# the missing element. `get(items, 99) + 1` does not type-check
# (reportOptionalOperand).
def get(items: list[int], index: int) -> int | None:
    if 0 <= index < len(items):
        return items[index]
    return None
